Move commands to seperate file test on robot soon

Rkortlandt opened the pull request:
Check on robot before merge

Rkortlandt committed:
Move commands to seperate file test on robot soon

deepsource-autofix[bot] committed:
style: format code with Google Java Format

This commit fixes the style issues introduced in f90bf4f according to the output
from Google Java Format.

Details: https://github.com/5152Alotobots/2024_Crescendo_Preseason/pull/13

SeanErn reviewed (approved):
Looks good, test it on the robot before merging.

Rkortlandt committed:
Add jsDoc

Rkortlandt committed:
Merge remote-tracking branch 'origin/seperateAutoCommands' into seperateAutoCommands

# Conflicts:
#	src/main/java/frc/robot/Auto.java

deepsource-autofix[bot] committed:
style: format code with Google Java Format

This commit fixes the style issues introduced in b7bde3c according to the output
from Google Java Format.

Details: https://github.com/5152Alotobots/2024_Crescendo_Preseason/pull/13

SeanErn reviewed (approved):
Good job with the javadocs, I missed that.

Rkortlandt committed:
put auto in the constructor bc/it was null otherwise
